libcxx/test/support/MinSequenceContainer.h

```diff
@@ -29,13 +29,23 @@ struct MinSequenceContainer {
   template <class It>
   explicit TEST_CONSTEXPR_CXX20 MinSequenceContainer(It first, It last) : data_(first, last) {}
   TEST_CONSTEXPR_CXX20 MinSequenceContainer(std::initializer_list<T> il) : data_(il) {}
+  template <class Range>
+  TEST_CONSTEXPR_CXX20 MinSequenceContainer(std::from_range_t, Range&& rg)
+      : data_(std::from_range, std::forward<Range>(rg)) {}
```

Comment:
It seems to me that this should be guard with `TEST_STD_VER >= 23` and unconditionally `constexpr`. Ditto `assign_range`.

No change requested as there's no new error, and perhaps we can do clean-up in the future. CI failures are unrelated.